notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<p>Great explanation!</p><p> </p><p>Nit: In the second format "(x_min,y_min,x_max,ymin,x_max,y_max,x_min,y_max)"</p><p>the third component should be "y_min", right? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>+1 I was wondering exactly this, what the numbers refer to. The explanation on the co-ordinates would be great.</p>